### ISSUE TITLE
Conda build issue

### ISSUE THREAD

**Issue by thomasmfish** [tlambert03/pycudadecon#55]
Hi, I just hit an issue using a conda-forge installed version of pycudadecon on Windows:
```
    from pycudadecon import RLContext, TemporaryOTF, rl_decon
.conda\Lib\site-packages\pycudadecon\__init__.py:18: in <module>
    from . import _libwrap as lib
.conda\Lib\site-packages\pycudadecon\_libwrap.py:24: in <module>
    lib = Library("libcudaDecon", version=_cudadecon_version)
.conda\Lib\site-packages\pycudadecon\_ctyped.py:31: in __init__
    self.lib = ctypes.CDLL(_file)
.conda\Lib\ctypes\__init__.py:366: in __init__
    if '/' in name or '\\' in name:
E   TypeError: argument of type 'NoneType' is not iterable
```
The error occurs when simply importing from pycudadecon (see the first line), and seems to be related to this: https://github.com/conda/conda-build/issues/4835. As mentioned in that issue, there is a workaround: https://github.com/conda/conda/issues/12563#issuecomment-1494264704.

I can always do a local build, so it shouldn't be a problem for me but I wanted to let you know!

**Comment by tlambert03** [comment]
ah, thanks very much for taking the time to let me know @thomasmfish!

**Comment by tlambert03** [comment]
should be fixed by #59
